fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -121,89 +133,77 @@ public String getLatestVersion(String url) {
             url = url.substring(0, index);
         }
 
-        return resourceType.cast(getResource(findResource(url, version), url, id));
+        return resourceType.cast(getResource(findRegistryResource(resourceType, url, version), url, id));
```

Comment:
worth adding `@throws ClassCastException` to the javadoc or you think its implied?

Reply:
added javadoc